releasenotes/notes/pulse-scheduler-8b259e2329baa21c.yaml

```diff
@@ -0,0 +1,67 @@
+---
+prelude: >
```

Comment:
Can you drop this section please. Having multiple prelude sections gets weird when we compile the rst for publishing. It basically becomes bullet points (like the other notes categories) for the intro section of the release. I've already got a prelude for 0.11.0 staged here: https://github.com/Qiskit/qiskit-terra/pull/3549 I'll migrate this over there

Reply:
https://github.com/Qiskit/qiskit-terra/pull/3549/commits/d08bb7b7cb401e464e27d7df4d0617f8df299d8a

Reply:
Fixed in 4cd817a5